runtime/runtime_options.go

```diff
@@ -20,22 +20,40 @@
 
 package runtime
 
-import "github.com/m3db/m3db/ratelimit"
+import (
+	"time"
+
+	"github.com/m3db/m3db/ratelimit"
+)
 
 const (
+	// defaultWriteNewSeriesAsync is false by default to give consistent
+	// writes by default
 	defaultWriteNewSeriesAsync = false
+
+	// defaultMaxWiredBlocks is 2MM by default which if using 2hr block sizes
+	// and writing every 10s at 1.4 point/byte (m3tsz) should use roughly 4gb:
+	// 1.4 * 6 * 120 * (2^21) = ~2gb
+	defaultMaxWiredBlocks = 2097152
```

Comment:
It's not bytes though, this is the max number of wired blocks we allow in process.